### PR TITLE
Added icherry alias to gallery

### PR DESCRIPTION
**Pull request by raiseandfall**
I added a simple ```icherry``` alias to cherrypick the changed files to stage to git. 
Sometimes you just want to go faster and not use ```git add --patch```.

### CODE REVIEW

File: gallery.sh [hunk 1]
```diff
@@ -14,3 +14,5 @@ function iseek() {
     iseek
 }
 
+# icherry: Interactive staging of selected changed files (faster than git add --patch)
+alias icherry='git status -s | sed s/^...// | ipt -m | git add'
```
Comment (ruyadorno) [inline gallery.sh]: hi @raiseandfall good idea! :smile: I was wondering why `icherry` as the command name and not `iadd` or something similar? I found it rather confusing as I was expecting `icherry` to be for actually cherry-picking something.

Here the only problem is that you have to convert data from the standard output into an argument at the end, to do so you would need to add `xargs` before `git add`

**Comment by ruyadorno** [comment]
Good idea, thanks for the PR :smile:

Can you please also add the same line to the end of the gallery on the README file?

**Comment by raiseandfall** [comment]
Hey @ruyadorno 

Indeed, I renamed icherry to iadd & added the code to the readme.

**Comment by ruyadorno** [comment]
Awesome :smile: can you just squash them into a single commit?

:+1: LGTM

**Comment by raiseandfall** [comment]
Done, thanks :+1:

**Comment by ruyadorno** [comment]
:confetti_ball: many thanks!

**Pull request merged**